packages/xchain-litecoin/src/utils.ts

```diff
@@ -112,26 +115,26 @@ export const getBalance = async (params: AddressParams): Promise<Balance[]> => {
   }
 }
 
-/**
- * Get the balance changes amount.
- *
- * @param {GetChangeParams} params
- * @returns {number} The change amount.
- */
-const getChange = async ({ valueOut, sochainUrl, network, address }: GetChangeParams): Promise<number> => {
-  try {
-    const balances = await getBalance({ sochainUrl, network, address })
-    const [ltcBalance] = balances.filter((balance) => assetToString(balance.asset) === assetToString(AssetLTC))
-    let change = 0
+// /**
```

Comment:
To be removed?

Reply:
yes, may as well since no one is currently calling it